snap/file.go

```diff
@@ -23,21 +23,20 @@ import (
 	"bytes"
 	"fmt"
 	"os"
+
+	"github.com/ubuntu-core/snappy/snap/squashfs" // to register it
```

Comment:
The comment is confusing. It's not that the import is doing some interesting side-effects any more. I'd drop the comment.